scrapy/utils/py36.py

```diff
@@ -1,10 +0,0 @@
-"""
-Helpers using Python 3.6+ syntax (ignore SyntaxError on import).
-"""
-
-
-async def collect_asyncgen(result):
```

Comment:
Maybe we need a deprecation for this.

I've renamed the module because it's useful to keep several things there even after we drop 3.5.

Reply:
Yes, we should deprecate it, just in case.